schema/atop_xslt.sch

```diff
@@ -13,63 +13,82 @@
     <!-- Global variables. -->
     <let name="docUri" value="document-uri(/)"/>
     
+    <let name="atopNamespaceUri" value="'http://www.tei-c.org/ns/atop'"/>
```

Comment:
I am leaning towards `http://www.tei-c.org/ns/program#atop`, but do not care much so long as a) the documentation (currently the wiki), b) this file, and c) whatever @martindholmes uses in the oXygen project are all the same.